projects/plugins/jetpack/_inc/client/search/dashboard/style.scss

```diff
@@ -46,12 +46,12 @@
 		top: 25rem;
 		min-height: 20rem;
 
-		background-color: #fff;
+		background-color: $white;
 		z-index: 1;
-
-		@media screen and ( min-width: 480px ) and ( min-height: 700px ) {
+		// In order to show WP footer, we need to sync the breakpoint.
+		@media screen and ( min-width: 782px ) and ( min-height: 700px ) {
 			top: 50vh;
-			height: 48vh;
+			height: calc( 48vh - 51px );
 		}
 	}
```

Comment:
Not directly related to this PR, but wanted to note this visual bug for viewports smaller than 878px. As is, the Instant Search preview clips below the footer.

<img width="545" alt="Screen Shot 2021-07-13 at 4 12 42 PM" src="https://user-images.githubusercontent.com/4044428/125531938-682361aa-10fa-4a99-80ce-52e450bd7012.png">

Reply:
>Also not directly related to this PR, but wanted to note this visual bug for viewports smaller than 878px. As is, the Instant Search preview clips below the footer.

Actually it is related to the break point change which tries to align the footer. Found absolute positioning is a bit hard for Responsive Web Design. Changed to relative which is way easier and doesn't need lot of media queries. The down side is that it's the toggle are not positioned perfectly per the design. 

Changed in f20d07947